osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/jms/MessageType.java

```diff
@@ -51,7 +51,7 @@
   GET_PROFILE_GENERIC_DATA,
   GET_SPECIFIC_ATTRIBUTE_VALUE,
   GET_STATUS,
-  GET_SUBSCRIPTION_INFORMATION,
+  GET_COMMUNICATION_NETWORK_INFORMATION,
```

Comment:
I'm afraid we need this because of the tight coupling of GXF messaging model to the SMHE recipe implementation. The Request and Response processors use these values.
This coupling still needs to be tackled.